### ISSUE TITLE
Tales Of Berseria (429660)

### ISSUE THREAD

**Issue by Zero86Sk** [ValveSoftware/Proton#827]
The game dosn't aunch at all.

My Steam Info
https://gist.github.com/Zero86Sk/d629f657c1ae861c36657cdf238a7765

here is the log
[steam-429660.log](https://github.com/ValveSoftware/Proton/files/2328339/steam-429660.log)

**Comment by JazzGlobal** [comment]
I looked through it and a few articles suggest disabling steam overlay. I also get this issue but I am not at my main PC so I can't test it right now. Please do! 

EDIT: Try the same for Symphonia too.

**Comment by Steveice10** [comment]
The Wine AppDB says this game doesn't run due to its Denuvo DRM.

**Comment by hagabaka** [comment]
I'd just like to add that disabling steam overlay doesn't help.

**Comment by zaps166** [comment]
- wine-staging 4.6, Steam for Windows - the game runs properly*.
- wine 4.6, Steam for Windows - the game freezes before it shows the window, CPU 100%.
- Proton 4.2-3 - the game still doesn't start.

\* only screen tearing is visible even with vsync ON, NVIDIA OSD shows "BLIT"

**Comment by Seizz** [comment]
As @zaps166, mentioned yes it does not even start. ;(
Yes staging wine is always progress thanks goodness... :dancer: 
So to sum @hagabaka, we can say that problem not in Steam Overlay at all...
The problem looks familiar to God Eater 1-2 a bit.
[steam-429660.log](https://github.com/ValveSoftware/Proton/files/3147909/steam-429660.log)

**Comment by Shin-Aska** [comment]
>     * wine-staging 4.6, Steam for Windows - the game runs properly*.
> 
>     * wine 4.6, Steam for Windows - the game freezes before it shows the window, CPU 100%.
> 
>     * Proton 4.2-3 - the game still doesn't start.
> 
> 
> * only screen tearing is visible even with vsync ON, NVIDIA OSD shows "BLIT"

I tried it with Wine 4.10, it works. Although it kinda feels awkward to maintain two different steam versions (Native and Windows)

**Comment by Vladimir-A** [comment]
Strange, Proton 4.11-1 - the game still doesn't start.

**Comment by Shin-Aska** [comment]
> Strange, Proton 4.11-1 - the game still doesn't start.

Use lutris and specifically, use the ge-protonified build to run a windows version of steam. It should work, along with other tales games such as Symphonia and Zestiria

**Comment by kust2708** [comment]
Works out of the box with Proton 5.0-1.

**Comment by TiZ-HugLife** [comment]
It now starts for me on Proton 5.0-1, but doesn't read controller input at the main menu. Controller input works fine with other games like SoulCalibur VI.

**Comment by kust2708** [comment]
@TiZ-EX1 
Controller input works for me with Steam Input (xinput) on.

**Comment by TiZ-HugLife** [comment]
My global controller settings have pretty much everything unchecked to make Steam Input as unobtrusive as possible. I tried to force Steam Input on in the per-game and it didn't work. But when I set it to Force Off, then it started working.

**Comment by adamluzsi** [comment]
thank you so much for making the game playable with Proton!

**Comment by KMP-7** [comment]
Game no longer works on Proton 5.0-4 for me. It was working earlier today using 5.0-3.

**Comment by CSahajdacny** [comment]
> Game no longer works on Proton 5.0-4 for me. It was working earlier today using 5.0-3.

Have you tried deleting the shadercache folder of the game?

**Comment by KMP-7** [comment]
> > Game no longer works on Proton 5.0-4 for me. It was working earlier today using 5.0-3.
> 
> Have you tried deleting the shadercache folder of the game?

I have not but that's the fix for Radeon cards right? I'm using an Nvidia card. I'm pretty sure this game uses Denuvo DRM and there was some kind of change done to get Just Cause 3 and Batman Arkham Knight (which also use Denuvo DRM) working in the latest version. My suspicion is that may have broken Tales of Berseria because it crashes just like it used to on Proton 4.11.

**Comment by CSahajdacny** [comment]
I use an Nvidia card too...  RTX 2060SUPER...

**Comment by KMP-7** [comment]
> I use an Nvidia card too... RTX 2060SUPER...

Does the game still work for you? If so where is that folder located?

**Comment by CSahajdacny** [comment]
.../steamapps/shadercache/SteamIDnumber
in this case will be .../steamapps/shadercache/429660

**Comment by KMP-7** [comment]
> .../steamapps/shadercache/SteamIDnumber
> in this case will be .../steamapps/shadercache/429660

Unfortunately it still crashes the same way during startup for me.

**Comment by kisak-valve** [comment]
Hello @gatx375, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by KMP-7** [comment]
> Hello @gatx375, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

Here's the log file, I had to compress it because it was larger than 10MB.
[steam-429660.log.zip](https://github.com/ValveSoftware/Proton/files/4315693/steam-429660.log.zip)

**Comment by Shin-Aska** [comment]
Can confirm, same experience as @gatx375 after upgrading to proton 5.0.4

**Comment by aeikum** [comment]
Thanks for reporting. We're able to reproduce and are looking into the problem.

**Comment by aeikum** [comment]
We have found a fix for this, which will be in the next Proton release. In the meantime, I have created a new app branch for the Proton 5.0 app called "proton-5.0-3" which you can use to play the game. You can find it on the BETAS tab in the settings for the Proton 5.0 tool in your Steam client.

**Comment by gpollo** [comment]
The game worked perfectly with `proton-5.0-3`. It seems that `proton-5.0-4` breaks the game again. It simply won't start.

**Comment by gpollo** [comment]
Just tested using a custom build from tag``proton-5.0-3` and sure enought, it works. Might bisect if I have time and the problem isn't trivial for others.

**Comment by aeikum** [comment]
@gpollo The problem is already understood, see comments above (it's the ntdll virtual memory commit).

**Comment by aeikum** [comment]
Hello. Tales of Berseria should be playable again in Proton 5.0-5. Thanks for reporting the problem!

**Comment by kust2708** [comment]
@aeikum It works for me, thanks !

**Comment by adamluzsi** [comment]
it works for me too, thank you for fixing it!

**Comment by roothorick** [comment]
There's a problem with extremely loud buzzing noises during FMVs that I suspect is specific to multichannel (i.e. surround sound) devices as I've observed an extremely similar issue in other games (e.g. Saints Row IV) and it seems the buzzing is only in the side, rear, and subwoofer (and sometimes center) channels.

Other games don't have the issue in Proton 4.11... but ToB doesn't run at all in 4.11.

**Comment by korodarn** [comment]
This game doesn't launch using 5.0-9, at least not on my system (Pop OS, kernel 5.6) 
[steam-429660.log.zip](https://github.com/ValveSoftware/Proton/files/4808674/steam-429660.log.zip)

**Comment by kisak-valve** [comment]
Hello @korodarn, does setting the game's launch options to `WINEDLLOVERRIDES=dxgi=n,b %command%` have an effect?

**Comment by korodarn** [comment]
@kisak-valve That did the trick, seems to work perfectly now. Thanks!

**Comment by nstgc** [comment]
For me the game launches and runs perfectly except for three annoyances:

First and foremost, in the anime cutscenes the game stops every 5 seconds. I need to defocus the game window in order to get it to play able, so I'm hitting the penguin (Windows) key on my keyboard to take me to the...overview mode (I'm not sure what it's called, but I'm using Gnome Shell).

This is a real problem for me since it breaks the feel of the game. Nothing like being at some dramatic part and then having it stop until you press a button.

I've tried switching from Borderless Window to Fullscreen, which leads to my second complaint. In Fullscreen mode the game doesn't completely defocus. I can change windows to another program, but I can't interact with it and the cursor remains that of the game (or so I suspect).

I've also tried fixing my issue with the cutscenes by using Proton-GE 5.9-2.

The third issue is that Velvet's nose looks *weird* and the fix is to use Special K...which doesn't work. This, of course, is not news.

**Comment by CygnusTerminal** [comment]
I have found two problems. On KDE Plasma (Arch Linux, i5 6300hq, GTX 960m, 12GB of DDR4 RAM, 240GB SSD+1 TB HDD) the game rendering would stop if i use the volume keys or try to alt tab. Sound still plays, but the game "freezes" for 10 seconds or so. Also, Alt+Tab doesn't work. The mouse keeps trying to focus the game's window, obviously making it nigh impossible to click on the desktop.
EDIT: To clarify, if ToB windows loses focus, it would freeze the entire machine for 10 seconds (sound still plays though) until becoming responsive again.

**Comment by CygnusTerminal** [comment]
[steam-429660.zip](https://github.com/ValveSoftware/Proton/files/4979805/steam-429660.zip)
Also @GloriousEggroll ToB does not launch as of Proton-5.9-GE-4-ST. Works fine with GE-3-ST. Log weighs over 370MB.

**Comment by ThisNekoGuy** [comment]
Works just fine with Proton GE 6 (I suspect it should with the latest regular Proton as well and it seemed that disabling ESYNC helps mitigate the cutscene juttering

It still does it but it's far less severe and doesn't require intervention

**Comment by boniek83** [comment]
Can't even run the game - crash on startup after downloading. Fedora 34/Xorg/Nvidia 470.63.01/Steam Flatpak, beta channel. Proton version doesn't seem to matter (whetver 6.16-GE-1 or 6.3-6 or Experimental)
[Tales of Berseria_d3d9.log](https://github.com/ValveSoftware/Proton/files/7111582/Tales.of.Berseria_d3d9.log)
Enabling/disabling MangoHUD doesn't make a difference.

**Comment by Shin-Aska** [comment]
@boniek83  just to rule everything out. Since you are using flatpak, are you using the Proton bundled via this instruction? https://github.com/flathub/com.valvesoftware.Steam/issues/642#issuecomment-716580163

While we are at it, can you give to us the output of the Proton log as well? If you don't know how. You can follow the instructions provided here https://github.com/ValveSoftware/Proton/wiki/Proton-FAQ#how-to-enable-proton-logs

**Comment by TiZ-HugLife** [comment]
Newer versions of Flatpak are capable of sub-sandboxing and don't require the specially-packaged build of Proton. You need Flatpak 1.11.1 or higher for it.

**Comment by boniek83** [comment]
Fedora 34 has flatpak 1.10, Fedora 35 will have flatpak 1.12. Flatpak 1.11 is basically a beta for 1.12.

> @boniek83 just to rule everything out. Since you are using flatpak, are you using the Proton bundled via this instruction? [flathub/com.valvesoftware.Steam#642 (comment)](https://github.com/flathub/com.valvesoftware.Steam/issues/642#issuecomment-716580163)
> 
> While we are at it, can you give to us the output of the Proton log as well? If you don't know how. You can follow the instructions provided here https://github.com/ValveSoftware/Proton/wiki/Proton-FAQ#how-to-enable-proton-logs

Thanks for pointing me in the right direction.  I was using proton straight from Steam so that might be a problem. I can't check now because it seems I have used up some sort of DRM limit trying to run game -__-. It works well with non-flatpak Steam, so I guess this is a non-issue here. If I will continue to have problems I will report it to flatpaks Steam. Thanks again.

**Comment by bl4ckout31** [comment]
I have noticed 2 annoying graphical glitches when the game is rendered in fullscreen or borderless windowed (normal windowed is fine).
Both of them appear momentarily right after opening or closing the start menu.
With the first one, the game shrinks slightly before going back to fullscreen. When this happens, I can see the taskbar at the bottom and the top of the window in the background or the top of my wallpaper.
With the second one, a residual frame from the graphics settings menu will appear for a few frames.

The first glitch will stop as soon as I change the screen mode, but then the second glitch appears instead.
I found that by keeping changing the screen mode between windowed and borderless windowed (or fullscreen) and spamming the start button, the second glitch also stops but it can be very tedious.

I have recorded a video showing everything as it is probably easier to understand what is happening this way : https://streamable.com/6sz9fm

Tested with Proton 6.3-6 and Proton 6.15-GE-2 on an up to date Archlinux with KDE 5.22.5 and Nvidia drivers 470.63.
I haven't really tried older Proton versions to avoid triggering Denuvo daily activation limit.

EDIT : Tried killing kwin before starting the game and no more glitches. I played Doom Eternal et Scarlet Nexus on the same setup (Proton included) with composition enabled and everything worked fine.

Proton logs : [steam-429660.log](https://github.com/ValveSoftware/Proton/files/7165243/steam-429660.log)

**Comment by KibaGasteiz** [comment]
The game was working great for me with Proton 6.3-6, but now it feels choppy with 6.3-7, it's something with the framerate that it's less fluid now.

This happened with Proton-GE and now something has been merged into official Proton, it may be something DXVK related, i don't really know.

**Comment by kisak-valve** [comment]
Hello @KibaGasteiz, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by KibaGasteiz** [comment]
[steam-429660.log](https://github.com/ValveSoftware/Proton/files/7274057/steam-429660.log)

PC INFO: https://gist.github.com/KibaGasteiz/f90ab0f1bd5509e4139fa99b79c1c067

Is there any easy way to have 6.3-6 so i can generate a log for it too?

**Comment by KibaGasteiz** [comment]
Trying Proton-6.18-GE from @GloriousEggroll and there is another issue with this game.
Sometimes when pressing controller d-pad directions it will count as pushing 2 times, so if you press d-pad down in a menu one time it will go 2 rows downs instead of 1, it doesn't happen everytime, more or less 50% of the times.
This may be related to the changes introduced about HID joystick thing, here's the log: 
[steam-429660.log](https://github.com/ValveSoftware/Proton/files/7274291/steam-429660.log)

Edit: Ok, be careful changing proton versions, installing and uninstalling the game and tinkering, the game uses denuvo and now thinks that i used it in so many PCs, I'm banned for 24 hours and can't play a game that I paid for. Thanks Bamco.

**Comment by alasky17** [comment]
@KibaGasteiz Could you check if the performance regression is a dxvk regression by dropping in an older version of dxvk (in this case 1.9.1 which was in 6.3-6: https://github.com/doitsujin/dxvk/releases/tag/v1.9.1)?  To do this: replace the dxvk files in the ~/.steam/steam/steamapps/common/Proton 6.3/dist/lib/wine/dxvk with the x32 dxvk files, and drop the x64 files in .../Proton 6.3/dist/lib64/wine/dxvk.  To get back to stock 6.3-7, just delete the Proton 6.3/dist folder and it will recreate itself.

**Comment by alasky17** [comment]
> Trying Proton-6.18-GE from @GloriousEggroll and there is another issue with this game. Sometimes when pressing controller d-pad directions it will count as pushing 2 times, so if you press d-pad down in a menu one time it will go 2 rows downs instead of 1, it doesn't happen everytime, more or less 50% of the times. This may be related to the changes introduced about HID joystick thing, here's the log: [steam-429660.log](https://github.com/ValveSoftware/Proton/files/7274291/steam-429660.log)

@KibaGasteiz Which type of controller are you using?  If you get a chance, could you test this when you are testing 6.3-7 with older dxvk to make sure you don't see the same issue there?

**Comment by KibaGasteiz** [comment]
> @KibaGasteiz Could you check if the performance regression is a dxvk regression by dropping in an older version of dxvk (in this case 1.9.1 which was in 6.3-6: https://github.com/doitsujin/dxvk/releases/tag/v1.9.1)? To do this: replace the dxvk files in the ~/.steam/steam/steamapps/common/Proton 6.3/dist/lib/wine/dxvk with the x32 dxvk files, and drop the x64 files in .../Proton 6.3/dist/lib64/wine/dxvk. To get back to stock 6.3-7, just delete the Proton 6.3/dist folder and it will recreate itself.

This doesn't seem to fix it, so it's not DXVK related, is there any way to get 6.3-6 again? Only found the source code and that was the best version for ToB, with that version i could test more because Proton5.13 has the same.. issue? i'm start to think that that fluid gameplay of 6.4-6 was the bug and the game looks choppy by default on Windows (never tried there and afraid of getting banned by denuvo again).

> @KibaGasteiz Which type of controller are you using? If you get a chance, could you test this when you are testing 6.3-7 with older dxvk to make sure you don't see the same issue there?

I'm using the XOne controller plugged through USB, but this bug only happens in Proton 6.18-**GE**-2, the gamepad works great on current Proton 5.3-7.
The latest version of Proton GE introduced some Wine changes for joysticks, so it may be just that, the new Wine version released today has more new implementations related to that so it may be fixed in the next GE version, who knows.

**Comment by alasky17** [comment]
@KibaGasteiz I uploaded 6.3-6 here with lock code "regression" so that you can test and see if performance really is better consistently on 6.3-6: https://www.codeweavers.com/xfer/alasky/proton-6.3-6/  You can untar that into ~/.steam/steam/compatibilitytools.d and reboot Steam.  

Please let me know if the gamepad is working better on the next GE version - the wine changes will eventually be in a stable Proton version as well, so catching regressions in Wine early is awesome :)

**Comment by CygnusTerminal** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/827#issuecomment-939354582

Game is indeed choppy on Windows without the fix from Kaldaien (he said it has to do with the way the FPS limiter works). I used to play with GE 5.9-ST-3 without issues (no choppiness, no drops, et cetera) aside the bakground sound being "off" (a lot of reverb). But now even that has problems on my hardware (Ryzen 5 4600h+GTX 1650) i reinstalled Arch quite a few times but i always have the same issue. Will test on Windows tomorrow to confirm if the choppiness is still present over there too.
EDIT: Tested on Windows, the game is a frametime mess, but not as bad as on Proton, jumping from 15.4 to 17.3ms as per MSI Afterburner, but no noticiable stutter/choppiness.

**Comment by CygnusTerminal** [comment]
So, i tested the game on Kubuntu and my issues are gone, gonna chuck this one to maybe a driver/xorg issue.

**Comment by KibaGasteiz** [comment]
> @KibaGasteiz I uploaded 6.3-6 here with lock code "regression" so that you can test and see if performance really is better consistently on 6.3-6: https://www.codeweavers.com/xfer/alasky/proton-6.3-6/ You can untar that into ~/.steam/steam/compatibilitytools.d and reboot Steam.
> 
> Please let me know if the gamepad is working better on the next GE version - the wine changes will eventually be in a stable Proton version as well, so catching regressions in Wine early is awesome :)

The gamepad issue is still happening with Proton 6.19-GE-2

And the stuttering issue is now happening with Proton 6.3-6, so it may be something with an update on the system, i'll keep testing everything i can, i'm just at the last stage and the game became unplayable for me. Bad luck i have.

**Comment by owallb** [comment]
> @bl4ckout31 Tried killing kwin before starting the game and no more glitches. I played Doom Eternal et Scarlet Nexus on the same setup (Proton included) with composition enabled and everything worked fine.

Thank you for this. I disabled compositing for the game in KDE Plasma by pressing Alt + F3 while having the game running, then selecting "More Actions"-> Configure Special Application Settings -> Add property -> Select "Block Compositing" and set it to "True" with "Force" selected. After this the most of the menu glitch is gone, it's still slightly glitchy but much better. 

I also noticed that Proton GE (6.18-GE-2 and 6.19-GE-2) makes it very laggy while if I switch back to Proton 6.3-7 the lag is gone.

**Comment by Fighter19** [comment]
I'm currently having troubles starting this game as well.

[steam-429660.log](https://github.com/ValveSoftware/Proton/files/7505566/steam-429660.log)

**Comment by SeongGino** [comment]
So is it safe to say that @bl4ckout31's KWin issue is still an observed bug? Is it a KDE or DXVK bug?
I'd like to know what's the proper recourse to report this to, because it's unreasonable to assume someone is expected to actively kill their WM just for two games. (I've also noticed this being an issue, albeit a smaller and more understandable one, with Thief Gold/TFix).

**Comment by CygnusTerminal** [comment]
> So is it safe to say that @bl4ckout31's KWin issue is still an observed bug? Is it a KDE or DXVK bug? I'd like to know what's the proper recourse to report this to, because it's unreasonable to assume someone is expected to actively kill their WM just for two games. (I've also noticed this being an issue, albeit a smaller and more understandable one, with Thief Gold/TFix).

I've also observed it on Warframe.

**Comment by SeongGino** [comment]
Update:
Can confirm that, whatever the source is, the shifting window geometry in KWin is a cause for (not-mod-related) crashes.

Recommend as a temporary measure, setting any KDE panels to Auto(Hide) when playing Berseria.

Doesn't seem to affect Zestiria as much, weirdly (though bug is still present there).

**Comment by Shin-Aska** [comment]
> Update: Can confirm that, whatever the source is, the shifting window geometry in KWin is a cause for (not-mod-related) crashes.
> 
> Recommend as a temporary measure, setting any KDE panels to Auto(Hide) when playing Berseria.
> 
> Doesn't seem to affect Zestiria as much, weirdly (though bug is still present there).

I am using KDE Plasma 5.12.9 (Which is the highest you can get from Ubuntu 18.04.6) and I tried reproducing the steps on @bl4ckout31's video. I can't replicate it. So if it is a bug somewhere, its possibly a KWin bug on a later version.

https://streamable.com/ohn95n

**Comment by SeongGino** [comment]
> > Update: Can confirm that, whatever the source is, the shifting window geometry in KWin is a cause for (not-mod-related) crashes.
> > Recommend as a temporary measure, setting any KDE panels to Auto(Hide) when playing Berseria.
> > Doesn't seem to affect Zestiria as much, weirdly (though bug is still present there).
> 
> I am using KDE Plasma 5.12.9 (Which is the highest you can get from Ubuntu 18.04.6) and I tried reproducing the steps on @bl4ckout31's video. I can't replicate it. So if it is a bug somewhere, its possibly a KWin bug on a later version.
> 
> https://streamable.com/ohn95n

So that isolates it pretty well; KWin's the culprit. Though unfortunately, 5.12 is *pretty* ancient and would need a bit more bisecting. :x 

And I'm on Arch; the lowest the Archives provides (without actually going to Archive.org lol) is 5.14. If anyone else could check other versions to see where this bug starts from, would be appreciated.

**Comment by Fighter19** [comment]
Regarding my "int $3" crash.

The game goes into an assertion when _vsnprintf from msvcr120.dll fails.

The actual problem appears to come from a too long path, that the game can't fit into it's variable.
I've remounted my drive using /mnt instead of /media/asdfghj/12345678-123-1234-1234-123456789abc
My files are under ./data/SteamLibrary (and so forth)
Now the game runs. It was a PITA to find out, imo. Definitely not something I expected.
I'm pretty sure others will run into this issue as well.
(e.g when launching the game of an SD card)

EDIT: I've tested this on Windows as well. It just seems to be a bug with this game.

**Comment by zaps166** [comment]
> I also noticed that Proton GE (6.18-GE-2 and 6.19-GE-2) makes it very laggy while if I switch back to Proton 6.3-7 the lag is gone.

I have screen tearing when V-Sync and/or FreeSync is enabled in game on Proton 7.0-6 and Proton Experimental.

It's running **correctly** with Proton 6.3-8.

I've copied DXVK DLLs from Proton 6.3-8 to Proton 7.0-6 and it didn't solve the issue.

The game is running on X11, single screen, fullscreen, X11 compositor is bypassed. NVIDIA OSD shows "BLIT" instead of "FLIP" - driver cant redirect the window with new Proton versions. Same on AMD Radeon GPUs (RADV).

`xwininfo` results:
[6.3-8.txt](https://github.com/ValveSoftware/Proton/files/10678723/6.3-8.txt)
[7.0-6.txt](https://github.com/ValveSoftware/Proton/files/10678722/7.0-6.txt)

**Comment by zaps166** [comment]
Hey @kisak-valve

It's very similar issue to https://github.com/ValveSoftware/Proton/issues/4200#issuecomment-1439047753

I can see the same log here: `vulkan:wine_vk_surface_set_offscreen Redirecting vulkan surface offscreen, expect degraded performance.`

**Comment by AJuujarvi** [comment]
>  I also noticed that Proton GE (6.18-GE-2 and 6.19-GE-2) makes it very laggy while if I switch back to Proton 6.3-7 the lag is gone.
> 
> I have screen tearing when V-Sync and/or FreeSync is enabled in game on Proton 7.0-6 and Proton Experimental.
> 
> It's running **correctly** with Proton 6.3-8.
> 
> I've copied DXVK DLLs from Proton 6.3-8 to Proton 7.0-6 and it didn't solve the issue.
> 
> The game is running on X11, single screen, fullscreen, X11 compositor is bypassed. NVIDIA OSD shows "BLIT" instead of "FLIP" - driver cant redirect the window with new Proton versions. Same on AMD Radeon GPUs (RADV).
> 
> `xwininfo` results: [6.3-8.txt](https://github.com/ValveSoftware/Proton/files/10678723/6.3-8.txt) [7.0-6.txt](https://github.com/ValveSoftware/Proton/files/10678722/7.0-6.txt)

Thank you for the report @zaps166! I was able to reproduce this issue so I can direct this to the right people. I was unable to reproduce [#4200](https://github.com/ValveSoftware/Proton/issues/4200#issuecomment-1439047753), so when the time comes that there is a fix for this game, I'll be asking if you can repro the issue still there as well.

**Comment by AJuujarvi** [comment]
@zaps166 We just released an update to Proton Experimental that should include a fix for the screen tearing. I am no longer able to reproduce it anymore at least. Can you confirm it is now fixed for you?

**Comment by zaps166** [comment]
@AJuujarvi 

I tried on `Proton: 1682527476 experimental-8.0-20230426`, screen tearing still visible. `41002.671:0128:012c:fixme:vulkan:wine_vk_surface_set_offscreen Putting vulkan surface back onscreen, expect standard performance.` - it's still in logs.

**Comment by AJuujarvi** [comment]
@zaps166 Looks like there was a mistake internally and it was not shipped out yet. Apologies. It should be in the next experimental release however now. Chances are it seemed fixed for me if the tearing was just on the edge of the screen. I checked again and that's what it looked like for me.

**Comment by alasky17** [comment]
@zaps166 It should now actually be shipped with experimental-8.0-20230505 :) Currently, the fix is gated to this game.  If you notice other games with the same issue, please tag me or @kisak-valve or @AJuujarvi on the game issue so that we can add the games to the hack :)

**Comment by zaps166** [comment]
@alasky17 Thanks, it's working now! However [this](https://github.com/ValveSoftware/Proton/issues/4200#issuecomment-1439047753) does not (I think it's similar issue).

**Comment by awaken1ng** [comment]
In case anyone else runs into input locking up when trying to save graphics settings, took me longer than it should've to realize that it was caused by resolution being empty.

So if you see this and you get stuck when trying to save:  
![image](https://github.com/user-attachments/assets/ce5a0fae-acb2-476e-a856-32b7d2db34e3)

Change it to have an actual value and try again.

**Comment by Raflos10** [comment]
This game fails to launch when I install it to an external drive, but it works on the internal drive.

[steam-429660.log](https://github.com/user-attachments/files/22089859/steam-429660.log)

**Comment by simifor** [comment]
@Raflos10 if you set this as the launch parameter, does the game work on the external drive? `PROTON_SET_GAME_DRIVE=1 %command%`

**Comment by Fighter19** [comment]
@Raflos10 The game has too small of an internal stack to keep track of long names.
It would crash on Windows as well. This is not a Wine issue.

**Comment by Raflos10** [comment]
> [@Raflos10](https://github.com/Raflos10) if you set this as the launch parameter, does the game work on the external drive? `PROTON_SET_GAME_DRIVE=1 %command%`

Yes that fixed it
